### PR TITLE
Release 2021-08-02

### PR DESCRIPTION
**Pull request by fisx**
## Checklist

 - [x] The **PR Title** explains the impact of the change.
 - [x] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
 - [x] If end-points have been added or changed: the **endpoint / config-flag checklist** (see Wire-employee only backend [wiki page](https://github.com/zinfra/backend-wiki/wiki/Checklists)) has been followed.
 - [x] If a schema migration has been added, I ran **`make git-add-cassandra-schema`** to update the cassandra schema documentation.
 - [x] Section *Unreleased* of **CHANGELOG.md** contains the following bits of information:
   - [x] A line with the title and number of the PR in one or more suitable sub-sections.
   - [x] If /a: measures to be taken by instance operators.
   - [x] If /a: list of cassandra migrations.
   - [x] If public end-points have been changed or added: does nginz need upgrade?
   - [x] If internal end-points have been added or changed: which services have to be deployed in a specific order?

### CODE REVIEW

**Comment by fisx** [comment]
NB: I snuck in ccee508, and linked it in the changelog rather than burying the information only in the latter.

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,6 +92,9 @@ This release requires a manual change in your galley configuration: `settings.co
 
 ## API Changes
 
+* A new team feature for classified domains is available (#1626):
+  - a public endpoint is at `GET /teams/:tid/features/classifiedDomains`
+  - an internal endpoint is at `GET /i/teams/:tid/features/classifiedDomains`
```
Comment (akshaymankar) [inline CHANGELOG.md:97]: Why did these things get added to previous release notes? Looks like a mistake.

Comment (fisx) [reply CHANGELOG.md]: e7d682d you're right, thanks!  it was like this on `develop`, but i wasn't careful enough to catch it.

**Review by akshaymankar — APPROVED** [review]
:rocket:

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,8 +37,57 @@
...
+
+Upgrade nginz (#1658)
+
+## API Changes
+
+## Features
+
+* A new team feature for classified domains is available (#1626):
+  - a public endpoint is at `GET /teams/:tid/features/classifiedDomains`
+  - an internal endpoint is at `GET /i/teams/:tid/features/classifiedDomains`
```
Comment (mdimjasevic) [inline CHANGELOG.md:57]: Should this have been mentioned in the API Changes section too?

Comment (fisx) [reply CHANGELOG.md:57]: you're right, that would have been better.  in my defense, it's an extension, not a change in existing end-points, and the changelog entry isn't too long to read it end-to-end, and it's already release, so i won't fix it this time.

**Pull request merged**